mods/seasonal/items.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 /**@type {{[k: string]: ModdedItemData}} */
+// @ts-ignore Hack for Snaquaza's Z-Move
```

Comment:
I accidentally resolved your comment @HoeenCoder but it is necessary because ModdedItemData has zMoveFrom defined as a string and not a string[], and even if all instances of the zMoveFrom itself are ts-ignored, this still gives off a type error because the object itself is incompatible

Reply:
I keep accidentally hitting resolve but alright, I’ll fix the definition.

Reply:
should be ts-ignoring that line then, not the entire ModdedItemData object.

Reply:
That’s where the error is though

Reply:
I dont get what error your getting, or why ts-ignoring the specific line where the error is (it would NOT be this line) wouldnt fix it. Can you show me it?